go.mod

```diff
@@ -116,7 +117,7 @@ require (
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/google/uuid v1.6.0
 	github.com/googleapis/gax-go/v2 v2.13.0
-	github.com/gorilla/websocket v1.5.1
+	github.com/gorilla/websocket v1.5.3
```

Comment:
Looks like this is not pre-release anymore:

* https://github.com/gorilla/websocket/releases/tag/v1.5.3